packages/x-components/src/x-modules/tagging/service/types.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Service to manipulate the session id.
+ *
+ * @public
+ */
+export interface SessionService {
+  /**
+   * Get the current session id from the storage:
+   * * If it's present then:
+   * * * If it's valid just return it.
+   * * * If it's outdated it will generate a new one.
+   * * If not creates a new one.
+   *
+   * @returns The session id.
+   */
```

Comment:
```suggestion
  /**
   * Get the current session id from the storage.
   *
   * @returns The session id.
   */
```

We are explaining what it's doing in the other file. I think that its enough to say that is returning you the session id.